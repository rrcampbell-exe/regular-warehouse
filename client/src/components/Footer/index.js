import React from "react";

const Footer = () => {

  return (
    <footer className='container-fluid text-center my-4'>
      <span className='text-center'>
        Regular Warehouse is brought to you by <a className='no-underline' href="https://github.com/rrcampbell-exe/regular-warehouse" target="blank">Regular Team</a>.
      </span>
    </footer>
  );
};

export default Footer;
