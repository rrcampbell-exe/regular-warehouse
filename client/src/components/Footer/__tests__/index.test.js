import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '..';

afterEach(cleanup);

describe('Footer component', () => {
    it('renders', () => { // this test passes
        render(<Footer/>)
    })
    it('matches snapshot', () => {
        const { asFragment } = render(<Footer/>);
        expect(asFragment()).toMatchSnapshot();
    })
})