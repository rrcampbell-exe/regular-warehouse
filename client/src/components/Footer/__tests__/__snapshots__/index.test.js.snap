// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer>
    <span>
      Regular Warehouse is brought to you by 
      <a
        href="https://github.com/rrcampbell-exe/regular-warehouse"
        target="blank"
      >
        Regular Team
      </a>
      .
    </span>
  </footer>
</DocumentFragment>
`;
